cloudsync/providers/filesystem.py

```diff
@@ -206,14 +209,20 @@ def empty(self):
 
     def on_any_event(self, event):
         """Called by watchdog on fs events."""
-        if type(event) == self.prev_event_type and event.src_path == self.prev_event_src and \
-                getattr(event, "dest_path", None) == self.prev_event_dest:
+
+        # filter out lots of the same events in the same millisecond
+        if (type(event) == self.prev_event_type and
+                event.src_path == self.prev_event_src and
+                getattr(event, "dest_path", None) == self.prev_event_dest and
+                time.monotonic() < self.prev_event_time + 0.01):
```

Comment:
technically, this code doesn't match the comment. This filters out events in the same 1/100th of a second